pkg/llms/zhipuai/api.go

```diff
@@ -124,3 +124,32 @@ func (z *ZhiPuAI) SSEInvoke(params ModelParams, handler func(*sse.Event)) error
 	}
 	return Stream(url, token, params, ZHIPUAI_MODEL_Default_Timeout, nil)
 }
+
+func (z *ZhiPuAI) Validate() error {
+	url := BuildAPIURL(ZhiPuAILite, ZhiPuAIAsyncInvoke)
+	token, err := GenerateToken(z.apiKey, API_TOKEN_TTL_SECONDS)
+	if err != nil {
+		return err
+	}
+
+	testPrompt := []Prompt{
+		{
+			Role:    "user",
+			Content: "Hello! I'm testing API, if you can receive this message, please reply 'Test success!'.",
```

Comment:
Will zhipuai really respond with `Test Success!` ?

Reply:
According to a test result:

> Hello! As an AI language model, I am capable of receiving and responding to text inputs, including messages like the one you are sending. However, I don't have any access to the internet or any specific API, so I don't have any information about whether the message was received successfully or not. Can you please provide more context or clarify your question?